practical/groovy/open_image.groovy

```diff
@@ -0,0 +1,102 @@
+/*
+ * -----------------------------------------------------------------------------
+ *  Copyright (C) 2018 University of Dundee. All rights reserved.
+ *
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * ------------------------------------------------------------------------------
+ */
+
+/*
+ * This Groovy script downloads a file and opens it in ImageJ using Bio-Formats
+ * exporter.
```

Comment:
exporter ? I would think importer, this would match with the line https://github.com/ome/training-scripts/pull/28/files#diff-66c94234b3208d29f85ff39a40086e75R93 - also, maybe worth hinting about when this script could be used (why not to open the image directly in ImageJ as the other scripts do ?)

Reply:
I have used the Jython scripts as a starting point
they both will need to be updated.
I think the script initially was trying to offer other options to open the images
I have since created more script but not pushed them yet

Reply:
Maybe just change the ``exporter`` to ``importer`` and we can deal with the more broad question later ?

Reply:
I think it is to show another simple way to open data
we show 3 different ways to read the image

Reply:
Agreed about the 3 different ways, that is I think beneficial.

Were you not telling me that you changed the work ``exporter`` here to ``importer`` (cannot find that commit) ?